cms/envs/common.py

```diff
@@ -2815,8 +2815,9 @@
 
 BRAZE_COURSE_ENROLLMENT_CANVAS_ID = ''
 
-DISCUSSIONS_INCONTEXT_FEEDBACK_URL = ''
-DISCUSSIONS_INCONTEXT_LEARNMORE_URL = ''
+# pylint: disable=line-too-long
+DISCUSSIONS_INCONTEXT_FEEDBACK_URL = "https://discuss.openedx.org/t/new-and-improved-discussions-forum/9183"
+DISCUSSIONS_INCONTEXT_LEARNMORE_URL = "https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/manage_discussions/discussions.html"
```

Comment:
Shouldn't this default to an openedx.readthedocs url instead of an edx.readthedocs URL?

Reply:
`openedx.readthedocs` shows a 404. I found this [doc](https://docs.openedx.org/en/latest/educators/migration_wip/17_manage_discussions/discussions.html) with`openedx` instead of `edx` but the **migration_wip** in the link indicates it is still a work in progress. 

If you have a link with `openedx`, please share.

Reply:
I see what you're saying. I guess the openedx docs aren't ready for this yet.